dataactcore/models/errorModels.py

```diff
@@ -43,8 +43,11 @@ class ErrorMetadata(Base):
     occurrences = Column(Integer)
     first_row = Column(Integer)
     rule_failed = Column(Text, nullable=True)
-    file_type_id = Column(Integer)
+    file_type_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_file_type_file_status_id"))
+    file_type = relationship("FileType", foreign_keys=[file_type_id])
     # Second file type id is used in cross file errors
-    target_file_type_id = Column(Integer)
-    original_rule_label = Column(Text, nullable = True)
-    severity_id = Column(Integer)
+    target_file_type_id = Column(Integer, ForeignKey("file_type.file_type_id", name="fk_target_file_type_file_status_id"))
+    target_file_type = relationship("FileType", foreign_keys=[target_file_type_id])
+    original_rule_label = Column(Text, nullable=True)
+    severity_id = Column(Integer, ForeignKey("rule_severity.rule_severity_id", name="fk_error_severity_id"))
+    severity = relationship("RuleSeverity")
```

Comment:
Thanks for adding these, it's good to start getting more use out of the consolidated DB.